bmds_server/common/templatetags/bs4.py

```diff
@@ -0,0 +1,28 @@
+"""
+Twitter Bootstrap 4 - helper methods
+"""
+from django import template
+
+register = template.Library()
+
+
+@register.tag(name="card")
+def bs4_card(parser, token):
```

Comment:
@casey1173 fyi; we might want to move this over to hawc for cards - see the screenshot in the PR